Neos.Neos/Classes/AssetUsage/Service/AssetUsageIndexingService.php

```diff
@@ -273,7 +273,7 @@ private function getAssetIdsByProperty(NodeType $nodeType, PropertyCollection $p
                     $propertyType,
                     $propertyValues->offsetGet($propertyName),
                 );
-            } catch (\Exception) {
+            } catch (\Throwable) {
```

Comment:
imo we should avoid catching `\Throwable` anywhere and just absorbing it as it would also silence any php error ... instead id say with the null check below we can revert that?

Reply:
good point. The exception should probably be caught within `extractAssetIds()` at the exact part(s) we want to catch it, e.g. line 302

Reply:
ill revert as written in https://github.com/neos/neos-development-collection/issues/5317 this pr implements both solutions and we only need one;) The null check will already be enough;)

see commit https://github.com/neos/neos-development-collection/commit/24d400fcdc567c9b2520171d63619149c987e386

Reply:
From my POV this is fine as it is, as we don't won't get bothered if just the fetching of assetids of a node makes problems.

Reply:
any problem must raise a \Exception, real php errors are dangerous to catch and ignore ... i thought that would be our practice^^

Reply:
I think throwable is just fine in some places. After all the real bad stuff is fatal and not even throwable, like memory, runtime, all syntax errors.